fiorella/src/styles/layout/_product.scss

```diff
@@ -0,0 +1,182 @@
+.tag-bold {
+  font-family: $merriweathersans;
+  font-weight: bold;
+
+  @media #{ $tablet }, #{ $desktop } {
+    padding-top: 10px;
+  }
+}
+
+.product-box {
+  border: 1px solid $cerise-red;
+  display: flex;
+  flex-direction: column;
+  margin: 10px 15px;
+
+  @media #{ $tablet }, #{ $desktop } {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .product-img-box {
+    width: 100%;
+  }
+
+  .product-info-box {
+    padding: 20px;
+    width: 100%;
+
+    .product-title {
+      display: none;
+      font-family: $playfair-display;
+      @media #{ $tablet }, #{ $desktop } {
+        display: block;
+      }
+    }
+
+    .product-description {
+      font-family: $merriweathersans;
+    }
+  }
+}
+
+.quantity-box {
+  font-family: $merriweathersans;
+  font-weight: bold;
+  margin-bottom: 20px;
+  margin-top: 20px;
+  margin-top: 0;
+
+  input {
+    font-family: $merriweathersans;
+    padding-right: 20px;
+    text-align: right;
+  }
+}
+
+.product-price-box {
+  margin: 20px 0;
+
+  .product-price {
+    color: $cerise-red;
+    font-family: $merriweathersans;
+    font-size: 30px;
+    font-weight: bold;
+  }
+
+  @media #{ $tablet }, #{ $desktop } {
+    margin: 20px 0;
+  }
+}
+.btn-submit {
+  margin: 20px 0;
+  text-align: center;
+
+  .btn-add-to-car {
+    background-color: $cerise-red;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets